### ISSUE TITLE
Cannot install using pip due to `python-sat` dependency

### ISSUE THREAD

**Issue by necrashter** [mvcisback/py-aiger-sat#5]
When I try to install this package using the following command:
```sh
pip install py-aiger-sat==3.0.6
```
It results in:
```
Collecting py-aiger-sat==3.0.6                                                                                                                                                                                     
  Using cached py_aiger_sat-3.0.6-py3-none-any.whl (6.2 kB)
Requirement already satisfied: py-aiger<7.0.0,>=6.0.0 in /home/ilker/miniconda3/lib/python3.10/site-packages (from py-aiger-sat==3.0.6) (6.2.2)
ERROR: Could not find a version that satisfies the requirement python-sat<0.2.0,>=0.1.0 (from py-aiger-sat) (from versions: 0.1.1.dev3, 0.1.1.dev4, 0.1.1.dev5, 0.1.1.dev6, 0.1.1.dev7, 0.1.1.dev8, 0.1.1.dev9, 0.1.2.dev0, 0.1.2.dev1, 0.1.2.dev2, 0.1.2.dev3, 0.1.2.dev4, 0.1.2.dev5, 0.1.2.dev6, 0.1.2.dev7, 0.1.2.dev8, 0.1.2.dev9, 0.1.3.dev0, 0.1.3.dev1, 0.1.3.dev2, 0.1.3.dev3, 0.1.3.dev4, 0.1.3.dev5, 0.1.3.dev6, 0.1.3.dev7, 0.1.3.dev8, 0.1.3.dev9, 0.1.3.dev10, 0.1.3.dev11, 0.1.3.dev12, 0.1.3.dev13, 0.1.3.dev14, 0.1.3.dev15, 0.1.3.dev16, 0.1.3.dev17, 0.1.3.dev18, 0.1.3.dev19, 0.1.3.dev20, 0.1.3.dev21, 0.1.3.dev22, 0.1.3.dev23, 0.1.3.dev24, 0.1.3.dev25, 0.1.4.dev0, 0.1.4.dev1, 0.1.4.dev2, 0.1.4.dev3, 0.1.4.dev4, 0.1.4.dev5, 0.1.4.dev6, 0.1.4.dev7, 0.1.4.dev8, 0.1.4.dev9, 0.1.4.dev10, 0.1.4.dev11, 0.1.4.dev12, 0.1.4.dev13, 0.1.4.dev14, 0.1.4.dev15, 0.1.4.dev16, 0.1.4.dev17, 0.1.4.dev18, 0.1.4.dev19, 0.1.4.dev20, 0.1.4.dev21, 0.1.4.dev22, 0.1.4.dev23, 0.1.4.dev24, 0.1.4.dev25, 0.1.5.dev1, 0.1.5.dev3, 0.1.5.dev5, 0.1.5.dev6, 0.1.5.dev7, 0.1.5.dev8, 0.1.5.dev9, 0.1.5.dev10, 0.1.5.dev11, 0.1.5.dev12, 0.1.5.dev13, 0.1.5.dev14, 0.1.5.dev15, 0.1.5.dev16, 0.1.5.dev17, 0.1.6.dev1, 0.1.6.dev2, 0.1.6.dev3, 0.1.6.dev4, 0.1.6.dev5, 0.1.6.dev6, 0.1.6.dev7, 0.1.6.dev8, 0.1.6.dev9, 0.1.6.dev10, 0.1.6.dev11, 0.1.6.dev12, 0.1.6.dev13, 0.1.6.dev14, 0.1.6.dev15, 0.1.6.dev16, 0.1.7.dev1, 0.1.7.dev2, 0.1.7.dev3, 0.1.7.dev4, 0.1.7.dev5, 0.1.7.dev6, 0.1.7.dev7, 0.1.7.dev8, 0.1.7.dev9, 0.1.7.dev10, 0.1.7.dev11, 0.1.7.dev12, 0.1.7.dev13, 0.1.7.dev14, 0.1.7.dev15, 0.1.7.dev16, 0.1.7.dev17, 0.1.7.dev18, 0.1.7.dev19, 0.1.7.dev20, 0.1.7.dev21, 0.1.7.dev22, 0.1.7.dev23, 0.1.7.dev24, 0.1.7.dev25, 0.1.7.dev26, 0.1.8.dev1, 0.1.8.dev2, 0.1.8.dev3, 0.1.8.dev4, 0.1.8.dev5, 0.1.8.dev6, 0.1.8.dev7, 0.1.8.dev8, 0.1.8.dev9, 0.1.8.dev10, 0.1.8.dev11, 0.1.8.dev12, 0.1.8.dev13, 0.1.8.dev14, 0.1.8.dev15, 0.1.8.dev16, 
0.1.8.dev17, 1.8.dev1, 1.8.dev2, 1.8.dev3)                                                                                                                                                                         
ERROR: No matching distribution found for python-sat<0.2.0,>=0.1.0
```
I guess the developers of `python-sat` decided to change its version number from `0.1.8` to `1.8`.

If I don't specify a version number, it tries to use an older version of this package and attempts to install `python-sat==0.1.5.dev13`:
```
Collecting py-aiger-sat                                                                                                                                                                                            
  Using cached py_aiger_sat-3.0.6-py3-none-any.whl (6.2 kB)                                                                                                                                                        
Requirement already satisfied: py-aiger-cnf<6.0.0,>=5.0.0 in /home/ilker/miniconda3/lib/python3.10/site-packages (from py-aiger-sat) (5.0.7)                                                                       
Requirement already satisfied: py-aiger<7.0.0,>=6.0.0 in /home/ilker/miniconda3/lib/python3.10/site-packages (from py-aiger-sat) (6.2.2)                                                                           
  Using cached py_aiger_sat-3.0.5-py3-none-any.whl (6.2 kB)                                                                                                                                                        
  Using cached py_aiger_sat-3.0.4-py3-none-any.whl (6.3 kB)                                                                                                                                                        
  Using cached py_aiger_sat-3.0.3-py3-none-any.whl (6.3 kB)                                                                                                                                                        
  Using cached py_aiger_sat-3.0.2-py3-none-any.whl (6.3 kB)                                                                                                                                                        
  Using cached py_aiger_sat-3.0.1-py3-none-any.whl (6.3 kB)                                                                                                                                                        
Collecting attrs<20.0,>=19.3                                                                                                                                                                                       
  Using cached attrs-19.3.0-py2.py3-none-any.whl (39 kB)                                                                                                                                                           Collecting py-aiger-sat                                                                                                                                                                                              Using cached py_aiger_sat-3.0.0-py3-none-any.whl (6.2 kB)                                                                                                                                                        Collecting python-sat==0.1.5.dev13                                                                                                                                                                                   Using cached python-sat-0.1.5.dev13.tar.gz (259 kB)
```
But installing `python-sat==0.1.5.dev13` fails due to `error: <urlopen error [Errno 110] Connection timed out>`. I presume the reason is that this version is really old and the links it depends on are broken.

Could you please look into this? Thanks.

**Comment by mvcisback** [comment]
Ah, thanks for letting me know.

I can take some time to bump the dependencies.

**Comment by mvcisback** [comment]
Should be fixed in `3.0.7`. Please reopen if you continue to have problems.
